src/components/Guilds/ProposalCard/index.tsx

```diff
@@ -109,10 +111,35 @@ interface ProposalCardProps {
 const ProposalCard: React.FC<ProposalCardProps> = ({ id, href }) => {
   const { guild_id: guildId } = useParams<{ guild_id?: string }>();
   const { data: proposal } = useProposal(guildId, id);
+  const votes = useVoteSummary(guildId, id);
   const { imageUrl, ensName } = useENSAvatar(
     proposal?.creator,
     DEFAULT_ETH_CHAIN_ID
   );
+  const [votesSummary, setVoteSummary] = useState([]);
+
+  // Build vote summary array
+  useEffect(() => {
+    const voteSummaryArray = [];
+
+    votes
+      .sort((a, b) => b - a)
+      .map((vote, i) => {
+        if (i < 3 && !(i === votes.length - 1)) {

```

Comment:
Maybe we should just set only the data and not the whole component to the state for performance reasons. Also, I think we can use `useMemo` instead of `useEffect` here.

What are we actually going to show if there are more than 2 options? Maybe something to discuss with Gero.